client-app/pages/sign-up/sign-up-page.vue

```diff
@@ -115,25 +142,40 @@ const { errors, handleSubmit, setFieldError } = useForm({
   validateOnMount: false,
 });
 
+const { value: registrationKind } = useField<RegistrationKind>("registrationKind");
 const { value: firstName } = useField<string>("firstName");
 const { value: lastName } = useField<string>("lastName");
 const { value: userName } = useField<string>("userName");
 const { value: email } = useField<string>("email");
+const { value: organizationName } = useField<string>("organizationName");
 const { value: password } = useField<string>("password");
 const { value: confirmPassword } = useField<string>("confirmPassword");
 
+const hasFormErrors = computed(() => !isObjectEmpty(errors.value));
 const commonErrors = ref<string[]>([]);
 
 const onSubmit = handleSubmit(async (data) => {
   commonErrors.value = [];
 
-  const result = await signMeUp({
-    email: `${data.email}`,
-    firstName: `${data.firstName}`,
-    lastName: `${data.lastName}`,
-    userName: `${data.userName}`,
-    password: `${data.password}`,
-  });
+  let result: IdentityResultType;
+  if (registrationKind.value == "personal") {
+    result = await registerUser({
+      email: `${data.email?.trim()}`,
```

Comment:
Maybe it's better to make a helper which returns such parsed string and use it instead?
Like `email: trimString(data.email)`